### PR TITLE
9 add context hook for products

### LINKED ISSUE #9: Add context hook for products.
- Add context hook for products display.
- Create custom hook for product context.
- Fetch data from server inside the context.

### PR DESCRIPTION
**Pull request by rdshinde**
- Added products context
- Fetched products from server.
- Displayed products.
- Added Loader to products page 
- fixed loader positioning

chrome-capture-2022-2-23

### CODE REVIEW

**Review by Bharatrathod-81 — APPROVED** [review]
looks good

File: src/components/display-item-card/displayItem.css [hunk 1]
```diff
@@ -53,3 +53,7 @@
 .card__img-container {
   padding: var(--scale-sm) var(--scale-xl);
 }
+.card__img-container img {
+  height: 420px !important;
+  width: 180px !important;
+}
```
Comment (raman1999) [inline src/components/display-item-card/displayItem.css:59]: try to avoid using !important

**Review by raman1999 — APPROVED** [review]
clean code written,  and UI looks good

File: src/components/display-item-card/displayItem.css (hunk 1 above)
Comment (rdshinde) [reply src/components/display-item-card/displayItem.css:59]: It's an exception for which I am not able to find a way. 
Thanks for pointing out.

**Pull request merged**